algebird-test/src/test/scala/com/twitter/algebird/DecayedValueSpec.scala

```diff
@@ -1,19 +1,30 @@
 package com.twitter.algebird
 
 import org.scalacheck.{ Arbitrary, Gen }
+import com.twitter.algebird.BaseProperties._
+import com.twitter.algebird.scalacheck.arbitrary._
+import org.scalacheck.Gen.choose
 import org.scalacheck.Prop._
 
-class DecayedValueLaws extends CheckProperties {
-  import org.scalacheck.Gen.choose
+class DecayedValueSpec extends CheckProperties {
```

Comment:
I was trying to make them all consistent - we have `Laws`, `Spec`, `Specification`, `Tests`... what do you think of `Spec` for everything? `Laws` is fine too, but I'd love them to match.